### PR TITLE
set sentry options from env vars

### PR DESCRIPTION
**Pull request by assertnotnull**

### CODE REVIEW

**Comment by gr2m** [comment]
Thanks for the pull request! Could you please update the documentation in the README? Ideally add links to Sentry's documentation in the description of each environment variable.

Once we merge this, we should add the same description to https://probot.github.io/docs/configuration/

File: README.md [hunk 1]
```diff
@@ -61,6 +61,9 @@ const log = pino(
...
 | `LOG_LEVEL_IN_STRING` | By default, when using the `json` format, the level printed in the log records is an int (`10`, `20`, ..). This option tells the logger to print level as a string: `{"level": "info"}`. Default `false` |
 | `SENTRY_DSN`          | Set to a [Sentry](https://sentry.io/) DSN to report all errors thrown by your app. <p>_(Example: `https://1234abcd@sentry.io/12345`)_</p>                                                                |
+| `SENTRY_ENV`          | Set the Sentry environment option. By default it's unset                                                                                                                                                 |
+| `SENTRY_DEBUG`        | Set the Sentry debug flag (use `true` to activate). By default it's false                                                                                                                                |
+| `SENTRY_RELEASE`      | Set the Sentry release option. By default it's unset                                                                                                                                                     |
```
Comment (gr2m) [inline README.md]: could you add links to the respective options in Sentry's documentation please? 

Just curious, what would be a use case to enable `SENTRY_DEBUG`?

Comment (assertnotnull) [reply README.md]: It's all in the documentation. I added a link to it in a more recent commit.

File: index.js [hunk 2]
```diff
@@ -25,6 +25,9 @@ function getTransformStream() {
       // See https://github.com/getsentry/sentry-javascript/issues/1964#issuecomment-688482615
       // 6 is enough to serialize the deepest property across all GitHub Event payloads
       normalizeDepth: 6,
+      environment: process.env.SENTRY_ENV,
+      debug: process.env.SENTRY_DEBUG === "true",
+      release: process.env.SENTRY_RELEASE,
```
Comment (gr2m) [inline index.js:30]: could you add tests for the additional configuration? Compare https://github.com/probot/pino/blob/03f0cf043bb3e29f4f0e32e79d7321dfb4dc850e/test/cli.test.js#L80-L136

**Pull request closed without merging**